package.json

```diff
@@ -103,7 +101,7 @@
     "react-native-permissions": "^3.0.1",
     "react-native-picker-select": "git+https://github.com/Expensify/react-native-picker-select.git#7f09b2c15ffae320d769788f75bdf8948714bb10",
     "react-native-plaid-link-sdk": "^7.2.0",
-    "react-native-reanimated": "3.0.0-rc.3",
+    "react-native-reanimated": "3.0.0-rc.6",
```

Comment:
Upgraded this because `3.0.0-rc.5` added jest 28 compatibility: https://github.com/software-mansion/react-native-reanimated/releases/tag/3.0.0-rc.5